<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1004</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_src">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>720</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>No Frame</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_yolo_det">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>530</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Detect</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_save_feature">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>530</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Feature</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_feature_name">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>535</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_save_file">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>580</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>no name</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_load_file">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>530</y>
      <width>160</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Files</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_crop">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>160</y>
      <width>224</width>
      <height>224</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string>No Crop</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_load_file">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>545</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>no name</string>
    </property>
   </widget>
   <zorder>pushButton_yolo_det</zorder>
   <zorder>label_src</zorder>
   <zorder>pushButton_save_feature</zorder>
   <zorder>lineEdit_feature_name</zorder>
   <zorder>label_save_file</zorder>
   <zorder>pushButton_load_file</zorder>
   <zorder>label_crop</zorder>
   <zorder>label_load_file</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1004</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
